### ISSUE TITLE
DockerJob TTY Error

### ISSUE THREAD

**Issue by fostiropoulos** [carlini/yet-another-applied-llm-benchmark#8]
Needless to say it is an amazing work.

I tried to extend ``DockerJob`` but failed as currently it does not seem to integrate with pty. 

Am I missing something or is this part of the code not tested yet?

**Comment by carlini** [comment]
Yeah this is definitely the flakiest part of the benchmark. It works on my linux machine, but I don't know if it will work well on other machines.

It would be great to re-work this entire piece of the code to be more ... good.
